### PR TITLE
46 login button landing page

### LINKED ISSUE #46: login button
Landing page should have a button allowing the user to login.

### PR DESCRIPTION
**Pull request by Sm3ggl3s**
closes #46 

# Problem
User has no way to login from landing page without manually entering the path within the url

# Solution
Added login button as well as sign up button to landing page that redirects user to dashboard once logged in.

Screenshot 2024-03-26 at 11 11 16

### CODE REVIEW

File: src/routes/+layout.server.ts [hunk 1]
```diff
@@ -3,13 +3,12 @@ import { loadFlash, setFlash } from "sveltekit-flash-message/server";
...
 import type { LayoutServerLoad } from "./$types";
+import { redirect } from "@sveltejs/kit";
 
-export const load: LayoutServerLoad = loadFlash(async ({ locals, cookies, fetch }) => {
-	let db:
-		| {
-				user: User;
-		  }
-		| undefined = undefined;
+export const load: LayoutServerLoad = loadFlash(async ({ locals, cookies, fetch, url }) => {
```
Comment (DylanHalstead) [inline src/routes/+layout.server.ts:8]: Lots of things being deleted here that we won. db should be undefined, there should still be the cookies.set stuff. Basically what was deleted add back and set it undefined

**Review by DylanHalstead — CHANGES_REQUESTED** [review]
Minor change once fixed lgtm

File: src/routes/+layout.server.ts (hunk 1 above)
Comment (Sm3ggl3s) [reply src/routes/+layout.server.ts:8]: I believe I fixed it. Can you just go in and check I did it correctly.

**Review by DylanHalstead — APPROVED** [review]
:kiss:

**Pull request merged**